packages/react-components/README.md

```diff
@@ -31,21 +32,28 @@ const testnet: Network = {
     jsonRpcUrl: 'https://json-rpc.testnet.concordium.com',
     ccdScanBaseUrl: 'https://testnet.ccdscan.io',
 };
-const walletConnectOpts: SignClientTypes.Options = { ... };
 
 function MyRootComponent() {
-    return (
-        <WithWalletConnector walletConnectOpts={walletConnectOpts} network={network}>
-            {(props) => <MyAppComponent {...props} />}
-        </WithWalletConnector>
-    );
+    return <WithWalletConnector network={network}>{(props) => <MyAppComponent {...props} />}</WithWalletConnector>;
 }
 
 function MyAppComponent(props: WalletConnectionProps) {
     // TODO Manage connections using the interface exposed through WalletConnectionProps (usually using useWalletConnectorSelector)...
 }
 ```
 
+Use `props.setActiveConnectorType(...)` and `props.connectActive()` from within `MyAppComponent`
+to set up a connector and initiate a connection.
```

Comment:
Did you consider also providing a react `Context` for doing these? Often times, you probably want to access these properties and actions provided by the `WithWalletConnector` component a little deeper than the `App` component, and it would save the user from having to set up a context themselves.

It's a "nice to have", but I thought I'd throw it out there 😉

Reply:
Nice idea, but I would find it strange to have two different ways of getting the same thing. So I actually consider it a feature that the user has to set it up themselves :)

The react docs also warn against overusing contexts, so we should only promote their use if there's a common need for it. At least in the simple dapps that I've worked on, it's my experience that these props are only needed one or two component levels down. I think it's reasonable that complex dapps would set up their own contexts if they need them.

Reply:
Didn't know they had those recommendations.. makes sense then 🙂